releasenotes/notes/primitives-v2-df871c0c6ac0b94a.yaml

```diff
@@ -0,0 +1,166 @@
+---
+features:
+  - |
+    Version 2 of the primitives was introduced via a new base class for both the sampler and the
+    estimator, along with new types for their inputs and outputs. The emphasis of this new
+    version is on performing vectorized calls to the primitive ``run()`` methods, so that sweeps
+    over parameter value sets and observables can be efficiently specified. See
+    :class:`~.StatevectorSampler` and :class:`~.StatevectorEstimator` for reference implementations
+    of the V2 primitives.
+
+    Moreover, the estimator has gained a ``precision`` argument in the :meth:`~.BaseEstimatorV2.run`
+    method that specifies the targeted precision of the expectation value estimates. Analogously,
+    the sampler has moved ``shots`` out of the options and into the arguments of the
+    :meth:`~.BaseSamplerV2.run` method. The sampler has also been changed to return the outputs
+    (e.g. bitstrings) from every shot, rather than providing a :class:`~.Counts`\-like return, and
+    also to store data from separate :class:`~.ClassicalRegister`\s . This enables derived classes
+    to implement sampler support for circuits with classical control flow.
+    
+    The primitive V2 base classes are:
+
+    * :class:`.BaseSamplerV2`
+    * :class:`.BaseEstimatorV2`
+
+    The new types which are used for inputs and outputs are:
+
+    * :const:`.SamplerPubLike`\: primitive unified bloc (PUB) of sampler inputs; a union type
+      of allowed inputs to a sampler
+    * :const:`.EstimatorPubLike`\: Primitive unified bloc (PUB) of estimator inputs; a union
+      type of allowed inputs to an estimator
+    * :class:`.PubResult`\: the data and metadata resulting from a single PUB's execution
+    * :class:`.DataBin`\: A namespace to hold data from a single PUB's execution
+    * :class:`.BitArray`\: an array-valued collection of bitstrings in a dense format
+    * :class:`.PrimitiveResult`: an iterable of :class:`.PubResult`\s along with metadata
+
+  - |
+    The reference implementation :class:`~.StatevectorEstimator` of :class:`~.BaseEstimatorV2` was
+    added. As seen in the example below, this estimator (and all V2 estimators) supports providing
+    arrays of observables and/or  arrays of parameter value sets that are attached to particular
+    circuits. 
+    
+    Each tuple of ``(circuit, observables, <optional> parameter values, <optional> precision)``,
+    called an estimator primitive unified bloc (PUB), produces its own array-based result. The
+    :meth:`~.EstimatorV2.run` method can be given many pubs at once. 
+
+    .. code-block:: python
```

Comment:
I'm not sure how this will render on the IBM website, but looking at the html in the Azure job the width is about 60 characters, so the lines here are too long and need scrollinge

<img width="727" alt="image" src="https://github.com/Qiskit/qiskit/assets/2235104/a8da832d-b4a9-4737-829d-7b7a21f49feb">